### PR TITLE
Fix species.param example doc and grammar

### PR DESCRIPTION
**Pull request by steindev**
Fix 1 grammar error and update(?)  example text for particle shape

### CODE REVIEW

**Comment by ax3l** [comment]
as always, please update all examples that have the same input file in `share/picongpu/examples/` as well... Please do as a follow-up PR.

File: include/picongpu/param/species.param [hunk 1]
```diff
@@ -43,7 +43,7 @@ namespace picongpu
  *  - particles::shapes::PCS : 3rd order
  *  - particles::shapes::P4S : 4th order
  *
- *  example:             using CICShape = particles::shapes::CIC;
+ *  example for using CIC shape: `using UsedParticleShape = particles::shapes::CIC;`
```
Comment (ax3l) [inline include/picongpu/param/species.param]: I also think this is not necessary. We write literally prosa here, now you say:

"example using CIC shape using CIC shape"... I would have stayed with `example:     using UsedParticleShape = particles::shapes::CIC;`

**Comment by psychocoderHPC** [comment]
@steindev You missed to fix the grammer `Define` -> `Defining` in all examples.

**Comment by steindev** [comment]
> @steindev You missed to fix the grammar `Define` -> `Defining` in all examples.

I fixed this in the follow-up PR #2795 as well.

**Pull request merged**